docs/index.mdx

```diff
@@ -28,7 +28,22 @@ import { RegionalUrl } from "@site/src/components/RegionalUrls";
 
 **LangSmith** is a platform for building production-grade LLM applications.
 It allows you to closely monitor and evaluate your application, so you can ship quickly and with confidence.
-LangChain's open source frameworks [langchain](https://python.langchain.com) and [langgraph](https://langchain-ai.github.io/langgraph/) work seemlessly with LangSmith but are not necessary - LangSmith works on its own!
+With LangSmith you can:
+- **Trace LLM Calls**: Gain visibility into LLM calls, and other parts of your application's logic. 
+- **Evaluate Performance**: Compare results across models and prompts to identify what works best.
```

Comment:
"models and prompts" feels a bit too narrow, maybe we include (agent/cognitive) architecture:
```suggestion
- **Evaluate Performance**: Compare results across models, prompts, and architectures to identify what works best.
```